Performing system checks...

System check identified no issues (0 silenced).
June 24, 2015 - 15:46:51
Django version 1.8.2, using settings 'my_blog.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
[24/Jun/2015 15:47:05]"GET /admin/ HTTP/1.1" 302 0
[24/Jun/2015 15:47:05]"GET /admin/login/?next=/admin/ HTTP/1.1" 200 1919
[24/Jun/2015 15:47:05]"GET /static/admin/css/base.css HTTP/1.1" 200 14049
[24/Jun/2015 15:47:05]"GET /static/admin/css/login.css HTTP/1.1" 200 940
[24/Jun/2015 15:47:05]"GET /static/admin/img/nav-bg.gif HTTP/1.1" 200 265
[24/Jun/2015 15:47:07]"POST /admin/login/?next=/admin/ HTTP/1.1" 200 2055
[24/Jun/2015 15:47:07]"GET /static/admin/img/icon_error.gif HTTP/1.1" 200 319
[24/Jun/2015 15:47:11]"POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[24/Jun/2015 15:47:11]"GET /admin/ HTTP/1.1" 200 3027
[24/Jun/2015 15:47:11]"GET /static/admin/css/dashboard.css HTTP/1.1" 200 434
[24/Jun/2015 15:47:11]"GET /static/admin/img/default-bg.gif HTTP/1.1" 200 836
[24/Jun/2015 15:47:11]"GET /static/admin/img/icon_addlink.gif HTTP/1.1" 200 119
[24/Jun/2015 15:47:11]"GET /static/admin/img/icon_changelink.gif HTTP/1.1" 200 119
[24/Jun/2015 15:47:17]"GET /admin/auth/group/ HTTP/1.1" 200 3599
[24/Jun/2015 15:47:17]"GET /static/admin/css/changelists.css HTTP/1.1" 200 5254
[24/Jun/2015 15:47:17]"GET /static/admin/js/core.js HTTP/1.1" 200 7522
[24/Jun/2015 15:47:17]"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 4464
[24/Jun/2015 15:47:17]"GET /static/admin/js/jquery.init.js HTTP/1.1" 200 326
[24/Jun/2015 15:47:17]"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 4464
[24/Jun/2015 15:47:17]"GET /static/admin/js/actions.js HTTP/1.1" 200 4910
[24/Jun/2015 15:47:17]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:47:17]"GET /static/admin/js/jquery.js HTTP/1.1" 200 284184
[24/Jun/2015 15:47:17]"GET /static/admin/img/icon_searchbox.png HTTP/1.1" 200 368
[24/Jun/2015 15:47:17]"GET /static/admin/img/nav-bg-reverse.gif HTTP/1.1" 200 178
[24/Jun/2015 15:47:17]"GET /static/admin/img/tooltag-add.png HTTP/1.1" 200 119
[24/Jun/2015 15:47:21]"GET /admin/auth/user/ HTTP/1.1" 200 6617
[24/Jun/2015 15:47:21]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:47:21]"GET /static/admin/img/icon-yes.gif HTTP/1.1" 200 299
[24/Jun/2015 15:47:21]"GET /static/admin/img/changelist-bg.gif HTTP/1.1" 200 50
[24/Jun/2015 15:47:21]"GET /static/admin/img/nav-bg-selected.gif HTTP/1.1" 200 265
[24/Jun/2015 15:47:21]"GET /static/admin/img/sorting-icons.gif HTTP/1.1" 200 369
[24/Jun/2015 15:47:24]"GET /admin/auth/user/1/ HTTP/1.1" 200 14145
[24/Jun/2015 15:47:24]"GET /static/admin/css/forms.css HTTP/1.1" 200 6283
[24/Jun/2015 15:47:24]"GET /static/admin/js/SelectBox.js HTTP/1.1" 200 4222
[24/Jun/2015 15:47:24]"GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 10127
[24/Jun/2015 15:47:24]"GET /static/admin/js/related-widget-wrapper.js HTTP/1.1" 200 857
[24/Jun/2015 15:47:24]"GET /static/admin/css/widgets.css HTTP/1.1" 200 10590
[24/Jun/2015 15:47:24]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:47:24]"GET /static/admin/js/calendar.js HTTP/1.1" 200 5969
[24/Jun/2015 15:47:24]"GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 17161
[24/Jun/2015 15:47:24]"GET /static/admin/img/icon_deletelink.gif HTTP/1.1" 200 181
[24/Jun/2015 15:47:24]"GET /static/admin/img/icon_calendar.gif HTTP/1.1" 200 192
[24/Jun/2015 15:47:24]"GET /static/admin/img/icon_clock.gif HTTP/1.1" 200 390
[24/Jun/2015 15:47:24]"GET /static/admin/img/icon-unknown.gif HTTP/1.1" 200 130
[24/Jun/2015 15:47:24]"GET /static/admin/img/selector-search.gif HTTP/1.1" 200 552
[24/Jun/2015 15:47:24]"GET /static/admin/img/selector-icons.gif HTTP/1.1" 200 2771
[24/Jun/2015 15:47:55]"GET /admin/ HTTP/1.1" 200 3027
[24/Jun/2015 15:47:57]"GET / HTTP/1.1" 200 1767
Performing system checks...

System check identified no issues (0 silenced).
June 24, 2015 - 15:47:02
Django version 1.8.2, using settings 'my_blog.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 312, in execute
    django.setup()
  File "/usr/local/lib/python3.4/dist-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.4/dist-packages/django/apps/registry.py", line 115, in populate
    app_config.ready()
  File "/usr/local/lib/python3.4/dist-packages/django/contrib/admin/apps.py", line 22, in ready
    self.module.autodiscover()
  File "/usr/local/lib/python3.4/dist-packages/django/contrib/admin/__init__.py", line 24, in autodiscover
    autodiscover_modules('admin', register_to=site)
  File "/usr/local/lib/python3.4/dist-packages/django/utils/module_loading.py", line 74, in autodiscover_modules
    import_module('%s.%s' % (app_config.name, module_to_search))
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2231, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2214, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2203, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1444, in exec_module
  File "<frozen importlib._bootstrap>", line 1549, in get_code
  File "<frozen importlib._bootstrap>", line 1509, in source_to_code
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/pavelwen/Program/my_blog/article/admin.py", line 2
    from article.models imprt Article
                            ^
SyntaxError: invalid syntax
Unknown command: 'sunserver'
Type 'manage.py help' for usage.
Unknown command: 'sunserver'
Type 'manage.py help' for usage.
Unknown command: 'sunserver'
Type 'manage.py help' for usage.
Unknown command: 'sunserver'
Type 'manage.py help' for usage.
Unknown command: 'sunserver'
Type 'manage.py help' for usage.
[24/Jun/2015 15:52:41]"GET /admin/ HTTP/1.1" 200 3681
[24/Jun/2015 15:52:41]"GET /static/admin/css/base.css HTTP/1.1" 304 0
[24/Jun/2015 15:52:41]"GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
[24/Jun/2015 15:52:41]"GET /static/admin/img/default-bg.gif HTTP/1.1" 304 0
[24/Jun/2015 15:52:42]"GET /static/admin/img/icon_addlink.gif HTTP/1.1" 304 0
[24/Jun/2015 15:52:42]"GET /static/admin/img/nav-bg.gif HTTP/1.1" 304 0
[24/Jun/2015 15:52:42]"GET /static/admin/img/icon_changelink.gif HTTP/1.1" 304 0
[24/Jun/2015 15:52:46]"GET /admin/article/article/ HTTP/1.1" 200 4358
[24/Jun/2015 15:52:46]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:52:49]"GET /admin/article/article/1/ HTTP/1.1" 200 5252
[24/Jun/2015 15:52:50]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:52:56]"GET /admin/article/article/1/ HTTP/1.1" 200 5252
[24/Jun/2015 15:52:56]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:53:03]"GET /admin/article/article/add/ HTTP/1.1" 200 5008
[24/Jun/2015 15:53:04]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:53:12]"GET /static/admin/img/default-bg-reverse.gif HTTP/1.1" 200 835
[24/Jun/2015 15:53:12]"POST /admin/article/article/add/ HTTP/1.1" 302 0
[24/Jun/2015 15:53:12]"GET /admin/article/article/ HTTP/1.1" 200 4737
[24/Jun/2015 15:53:12]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:53:12]"GET /static/admin/img/icon_success.gif HTTP/1.1" 200 341
[24/Jun/2015 15:53:17]"GET /admin/article/article/1/ HTTP/1.1" 200 5252
[24/Jun/2015 15:53:17]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
Performing system checks...

System check identified no issues (0 silenced).
June 24, 2015 - 15:52:39
Django version 1.8.2, using settings 'my_blog.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
[24/Jun/2015 15:58:48]"GET /admin/article/article/ HTTP/1.1" 200 9788
[24/Jun/2015 15:58:48]"GET /admin/jsi18n/ HTTP/1.1" 200 2344
[24/Jun/2015 15:58:48]"GET /static/bootstrap_admin/js/base.js HTTP/1.1" 200 3521
[24/Jun/2015 15:58:48]"GET /static/admin/css/base.css HTTP/1.1" 200 6445
[24/Jun/2015 15:58:48]"GET /static/admin/css/changelists.css HTTP/1.1" 200 4437
[24/Jun/2015 15:58:48]"GET /static/bootstrap_admin/css/bootstrap.min.css HTTP/1.1" 200 113498
[24/Jun/2015 15:58:48]"GET /static/admin/js/core.js HTTP/1.1" 304 0
[24/Jun/2015 15:58:48]"GET /static/admin/js/jquery.js HTTP/1.1" 304 0
[24/Jun/2015 15:58:48]"GET /static/bootstrap_admin/js/bootstrap.min.js HTTP/1.1" 200 35713
[24/Jun/2015 15:58:48]"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
[24/Jun/2015 15:58:48]"GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
[24/Jun/2015 15:58:48]"GET /static/admin/js/jquery.js HTTP/1.1" 304 0
[24/Jun/2015 15:58:48]"GET /static/admin/js/actions.js HTTP/1.1" 304 0
[24/Jun/2015 15:58:48]"GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
[24/Jun/2015 15:58:48]"GET /static/bootstrap_admin/fonts/glyphicons-halflings-regular.woff HTTP/1.1" 200 23320
[24/Jun/2015 15:58:53]"GET /admin/ HTTP/1.1" 200 10937
[24/Jun/2015 15:58:54]"GET /static/bootstrap_admin/js/dashboard.js HTTP/1.1" 200 18477
[24/Jun/2015 15:59:00]"GET /admin/ HTTP/1.1" 200 10937
[24/Jun/2015 15:59:00]"GET /static/bootstrap_admin/js/bootstrap.min.js HTTP/1.1" 304 0
[24/Jun/2015 15:59:00]"GET /static/admin/css/base.css HTTP/1.1" 304 0
[24/Jun/2015 15:59:00]"GET /static/bootstrap_admin/css/bootstrap.min.css HTTP/1.1" 304 0
[24/Jun/2015 15:59:00]"GET /static/bootstrap_admin/js/base.js HTTP/1.1" 304 0
Performing system checks...

System check identified no issues (0 silenced).
June 24, 2015 - 15:58:01
Django version 1.8.2, using settings 'my_blog.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
June 24, 2015 - 16:00:08
Django version 1.8.2, using settings 'my_blog.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
June 24, 2015 - 16:00:42
Django version 1.8.2, using settings 'my_blog.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
June 24, 2015 - 16:00:58
Django version 1.8.2, using settings 'my_blog.settings'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 303, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2231, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2214, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2203, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1448, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/pavelwen/Program/my_blog/my_blog/settings.py", line 110, in <module>
    'django.core.context_processors.request')
TypeError: can only concatenate tuple (not "str") to tuple
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 303, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2231, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2214, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2203, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1448, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/pavelwen/Program/my_blog/my_blog/settings.py", line 109, in <module>
    ('django.core.context_processors.request')
TypeError: can only concatenate tuple (not "str") to tuple
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 303, in execute
    settings.INSTALLED_APPS
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2231, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2214, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2203, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1448, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/home/pavelwen/Program/my_blog/my_blog/settings.py", line 109, in <module>
    'django.core.context_processors.request'
TypeError: can only concatenate tuple (not "str") to tuple
